<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUIT</class>
 <widget class="QDialog" name="QUIT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1488</width>
    <height>800</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="TEXT">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>149</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>6</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frameRight">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>77</y>
     <width>1211</width>
     <height>550</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="processedImgLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CAMERA">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>729</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAPTURE">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>729</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UNDO">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>650</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UPLOAD">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>650</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Upload Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UNDO_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>650</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Select">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>650</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Select Excel File</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalObjectsLabel">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Objects: 0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>730</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RELOAD">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>650</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UNDO_4">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>650</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <zorder>frameRight</zorder>
  <zorder>TEXT</zorder>
  <zorder>imgLabel</zorder>
  <zorder>CAMERA</zorder>
  <zorder>CAPTURE</zorder>
  <zorder>UNDO</zorder>
  <zorder>UPLOAD</zorder>
  <zorder>UNDO_2</zorder>
  <zorder>Select</zorder>
  <zorder>totalObjectsLabel</zorder>
  <zorder>comboBox</zorder>
  <zorder>RELOAD</zorder>
  <zorder>UNDO_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
